api/jobs.py

```diff
@@ -40,132 +42,124 @@
 def valid_transition(from_state, to_state):
     return (from_state + ' --> ' + to_state) in JOB_TRANSITIONS or from_state == to_state
 
-Category = util.Enum('Category', {
-    'classifier': 'classifier', # discover metadata
-    'converter':  'converter',  # translate between formats
-    'qa':         'qa',         # quality assurance
-    'analytical': 'analytical', # general purpose
-})
-
-Gear = namedtuple('gear', ['name', 'category', 'input'])
-
 def get_gears():
     """
     Fetch the install-global gears from the database
     """
 
     gear_doc  = config.db.static.find_one({'_id': 'gears'})
-    gears_map = gear_doc['gear_list']
-    gears = []
+    return gear_doc['gear_list']
 
-    for g in gears_map:
-        cat_string = g['category']
-        gears.append(Gear(g['name'], Category[cat_string], g['input']))
+def get_gear_by_name(name):
 
-    return gears
+    # Find a gear from the list by name
+    gear_doc = config.db.static.find_one({
+        "_id": "gears",
+        "gear_list": {
+            "$elemMatch": {
+                "name": name
+            }
```

Comment:
I tried that later query and I got this:

```javascript
({'_id': 'gears'}, {'gear_list.name': 'dcm_convert'}) ->

{
    "_id" : "gears",
    "gear_list" : [ 
        {
            "name" : "dicom_mr_classifier"
        }, 
        {
            "name" : "dcm_convert"
        }, 
        {
            "name" : "qa-report-fmri"
        }
    ]
}
```

I instead want the whole gear subdoc:

```javascript
{'_id': 'gears'}, {'gear_list': {'$elemMatch': {'name': 'dcm_convert'}}}

{
    "_id" : "gears",
    "gear_list" : [ 
        {
            "name" : "dcm_convert",
            ...
            "manifest" : {
                 ...
            }
        }
    ]
```